mojo/public/cpp/platform/tcp_platform_handle_utils_posix.cc

```diff
@@ -76,19 +69,23 @@ base::ScopedFD CreateTCPClientHandle(uint16_t port) {
     unix_addr.sin_addr.s_addr = inet_addr("127.0.0.1");
   unix_addr_len = sizeof(struct sockaddr_in);
 
-  base::ScopedFD handle = CreateTCPSocket(false, IPPROTO_TCP);
-  LOG(INFO) << "Client Sock fd for port " << port <<":"<<handle.get();
-  if (handle.get() < 0)
-    return base::ScopedFD();
-  if (HANDLE_EINTR(connect_retry(handle.get(),
-      reinterpret_cast<sockaddr*>(&unix_addr), unix_addr_len)) < 0) {
-    PLOG(ERROR) << "connect " << handle.get();
-    return base::ScopedFD();
+  PlatformHandle handle = CreateTCPSocket(false, IPPROTO_TCP);
+  if (!handle.is_valid())
+    return PlatformHandle();
+
+  if (HANDLE_EINTR(connect_retry(handle.GetFD().get(),
```

Comment:
need this?

Reply:
Yes, I think this is needed. CreateTCPSocket() can return empty PlatformHandle.